public/app/plugins/datasource/cloudwatch/components/LogsQueryField.tsx

```diff
@@ -32,6 +44,10 @@ const rowGap = css`
   gap: 3px;
 `;
 
+const addPaddingToButton = css`
+  padding: 1px 4px;
+`;
+
```

Comment:
Can this be removed? Looks like it's getting overridden anyway. It still looks the same as before without the padding.

<img width="1273" alt="Screen Shot 2022-10-19 at 11 31 32 AM" src="https://user-images.githubusercontent.com/19530599/196775121-8ee235d0-5cd2-459d-8a0f-a80e39086fc6.png">

Reply:
@kevinwcyu I used `cx` instead of `classNames` and it is fixed 🙂